src/w3c/templates/cgbg-headers.js

```diff
@@ -130,8 +129,8 @@ export default conf => {
                     ${[conf.additionalCopyrightHolders]} &amp;
                   `
                 : ""}
-              the Contributors to the ${conf.title} Specification, published by
-              the
+              the Contributors to the ${document.title} Specification, published
```

Comment:
This needs to be a clone of the children of `getSpecTitleElem()`. Otherwise, we lose the `<code>` or other tags.